c7n/resources/securityhub.py

```diff
@@ -284,18 +284,20 @@ class PostFinding(Action):
     schema = type_schema(
         "post-finding",
         required=["types"],
-        title={"type": "string"},
-        description={'type': 'string'},
+        title={"type": "string", 'default': 'policy.name'},
+        description={'type': 'string', 'default':
+            'policy.description, or if not defined in policy then policy.name'},
         severity={"type": "number", 'default': 0},
         severity_normalized={"type": "number", "min": 0, "max": 100, 'default': 0},
         confidence={"type": "number", "min": 0, "max": 100},
         criticality={"type": "number", "min": 0, "max": 100},
         # Cross region aggregation
-        region={'type': 'string', 'description': 'cross-region aggregation target'},
+        region={'type': 'string', 'description': 'cross-region aggregation target',
+            'default': 'current region'},
```

Comment:
hmm.. default of current region and the description don't quite logically go together. default none or no-aggregation would be better.